### PR TITLE
DevWarning: fix incorrect link to live WebStudio

### PR DESCRIPTION
**Pull request by markspolakovs**

### CODE REVIEW

File: src/showplanner/PISModal.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ function DevWarning() {
   if (process.env.REACT_APP_PRODUCTION === "true") {
     return null;
   }
+  const wsUrl =
+    process.env.REACT_APP_MYRADIO_NONAPI_BASE + "/MyRadio/webstudio";
```
Comment (mstratford) [inline src/showplanner/PISModal.tsx:12]: I presume this is the redirect thing? It feels a bit weird to have u.o.u/myradio/MyRadio/webstudio

Comment (markspolakovs) [reply src/showplanner/PISModal.tsx:12]: Yep, it is (see https://ury.org.uk/myradio/MyRadio/webstudio/?ref=WebStudio)

Comment (mstratford) [reply src/showplanner/PISModal.tsx:12]: That just looks weird :/

Comment (mstratford) [reply src/showplanner/PISModal.tsx:12]: But whatever.

**Pull request merged**